packages/core/lib/parser/abiParser.spec.ts

```diff
@@ -101,3 +103,17 @@ describe("parseEvent", () => {
     });
   });
 });
+
+describe("parse", () => {
+  describe("fallback functions", () => {
+    it("should work on output-less fallback functions", () => {
+      const fallbackAbiFunc: RawAbiDefinition = {
+        payable: true,
+        stateMutability: "payable",
+        type: "fallback",
+      } as any;
```

Comment:
Had to mark this as `any` because it doesnt conform to `RawAbiDefinition`. The interface needs to have more properties marked as optional but it was out of scope for the fix we need